andela/apollo-ah-backend
# #166376212  Fix email notification malfunctionality.

#### What does this PR do?
It fixes email notification malfunctionality.

#### Description of Task to be completed?
- change email notification delivery service from nodemailer to sendgrid
- fix malfunctionality of account confirmation email notification for users
- fix malfunctionality of reset password email notification for users

#### How should this be manually tested?
- Follow the steps below:
  - Run the following commands in your terminal
`git clone https://github.com/andela/apollo-ah-frontend.git`
- Open the project folder `cd apollo-ah-frontend` in your terminal
- Run `npm install` to install the project dependencies.
- Open the project folder in your `text editor` of choice.
- Create a `.env` file in the root directory.
- Locate `.env.example` file in the root directory and copy those environment variables keys in that file to `.env` file.
- Provide environment variable values to those keys you copied to `.env` file.
- Run `npm start:dev` to start the app.
- Open up the app in your browser of choice.

`For reset password email notification`
- Navigate to Login page and click on the `Forgot Password`.
- Input your email address on the reset password form and request to reset your password.
- Move over to your your email
- You should be to see an email for resetting your password

`For account confirmation email notification`.
- Navigate to sign up page and sign up with your details.
- Move over to your your email
- You should be to see an for account confirmation
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166376212](https://www.pivotaltracker.com/story/show/166376212)

#### Screenshots (if appropriate)
N/A

#### Questions:
